README.md

```diff
@@ -10,14 +10,13 @@ npm install -g bitcore-node
 bitcore-node start
 ```
 
-Note: For your convenience, we distribute binaries for x86_64 Linux and x86_64 Mac OS X. Upon npm install, the binaries for your platform will be downloaded. For more detailed installation instructions, or if you want to compile the project yourself, then please see the [Build & Install](docs/build.md) documentation to build the project from source.
+Note: For your convenience, we distribute bitcoind binaries for x86_64 Linux and x86_64 Mac OS X. Upon npm install, the binaries for your platform will be downloaded. For more detailed installation instructions, or if you want to compile the project yourself, then please see the Bitcore branch of [Bitcoin Core with additional indexing](https://github.com/bitpay/bitcoin/tree/0.12-bitcore).
 
 ## Prerequisites
 
 - Node.js v0.12 or v4.2
-- ~100GB of disk storage
+- ~150GB of disk storage
 - ~4GB of RAM
-- Mac OS X >= 10.9, Ubuntu >= 12.04 (libc >= 2.15 and libstdc++ >= 6.0.16)
 
```

Comment:
There is a "gotcha" that exists on Debian systems. Even after you npm install bitcore or npm install bitcore-node: you must have build-essential and, this is the gotcha, you must have libzmq3-dev and you must apt-get remove libzmq1. If you don't you get "Uncaught Error: Monitoring support disabled check zmq version is > 3.2.1 and recompile this addon". You also need to ensure your bitcoin.conf has something similar to:

spentindex=1
addressindex=1
txindex=1
whitelist=127.0.0.1
server=1
rpcuser=bitcoin
rpcpassword=local321
zmqpubrawtx=tcp://127.0.0.1:3002
zmqpubhashblock=tcp://127.0.0.1:3002

Note that the port for the zmq* options should NOT be the same port as you are serving insight-api requests. This tripped me up a bit when I was forming the puzzle pieces in my head. Please correct me if I am not correct.